{{Stony_Brook}}
<html>
  <head>
    <style>
      h1.ui.header {
        padding: 2.5em 0em 1.5em;
        font-size: 4em;
      }

      h2.ui.header {
        padding: 0.5em 0em;
        font-size: 1.5em;
      }

      #HQ_page p.black {
        font-size: 1em;
        color: #444444;
      }
      
      div.image {
          margin-bottom: 1em;
      }

      /* Mobile */
      @media only screen and (min-width: 300px) and (max-width: 767px){
        img.ui.image.section {
          max-width: 40%;
        }

        #HQ_page p.black, li {
          font-size: 1.2em;
        }

        div.ui.text.container {
          padding-left: 2em;
          padding-right: 2em;
        }
      }
    </style>
  </head>
  <body>
    <!-- Header -->
    <div class="ui fluid container" style="background-color: #05386b">
      <div data-aos="zoom-out" data-aos-delay="0" class="ui center aligned container">
        <h1 class="ui inverted header">
          Design
        </h1>
      </div>
    </div>

    <!-- Lab and Equipment -->
    <div class="ui text container" style="padding-bottom: 2.5em">
      <h2 class="ui header">Expression of yeast XRN1p in <i>Nicotiana benthamiana</i></h2>
      <p class="black">
        For our project it would not have been feasible to produce completely transgenic plants that were expressing XRN1p in the timeframe we were working in. So we settled on providing a proof of concept through transient expression in <i>N. benthamiana</i> leaves using Agrobacterium mediated DNA transfer. This required propagation of plasmids containing our construct in <i>E.coli</i>, and Agrobacterium followed by an infiltration into <i>N. benthamiana</i>.
      </p>

      <div class="image">
        <img class="ui centered image" src="https://2019.igem.org/wiki/images/3/31/T--Stony_Brook--design-2.jpg">
      </div>

      <p class="black">
        We chose to work with pMY200, an empty backbone containing 25 bp LB and RB T-DNA repeats which could be recognized by the vir gene on the helper plasmid in our agrobacterium to insert our construct into the plantâ€™s genome. There is an origin of replication for both <i>E.coli</i> and Agrobacterium allowing the plasmid to stably exist in both organisms. The KanR gene confers resistance to Kanamycin, which was used to select for transformed DH5-alpha competent <i>E.coli</i> and Agrobacterium.
      </p> <br>

      <p class="black">
        Since our vector did not contain a promoter or terminator, we chose to put the XRN1 gene under the control of the 35s CaMV promoter and NOS terminator. These regulatory elements are commonly used in plant synthetic biology and would allow for strong expression of our genes in the <i>N. benthamiana</i> leaves. In addition we fused mCherry, a red fluorescent protein to the C terminus of XRN1, so that we could visually monitor the expression of our genes in the plant leaves.
      </p> <br>

      <p class="black">
        We utilized Gibson Assembly to insert our constructs into pMY200. This required us to design overlaps between adjacent DNA sequences. Overlaps of of 15-25 base pairs were added to each part prior to synthesis. Due to the size of the XRN1 gene, the synthesis needed to be carried out in two separate pieces of about 2300 bp each. In total 5 pieces were assembled into our vector: A 35s promoter, 2 fragments of the XRN1, an mCherry fusion protein, and a NOS terminator. The overlaps were oriented such that the 35s promoter would be adjacent to the RB T-DNA repeat to prevent degradation of our construct inside the plant tissue. The order after assembly is RB Repeat - 35s Promoter - XRN1 - mCherry - NOS Terminator - LB Repeat.
      </p> <br>

      <div class="image">
        <img class="ui centered image" src="https://2019.igem.org/wiki/images/d/d7/T--Stony_Brook--design-3.jpg">
      </div>

      <p class="black">
        In order to observe the effect of our construct, agrobacterium containing our construct will be infiltrated into <i>N. benthamiana</i> leaves that are co-infected with TMV. We plan on collecting data on mCherry fluorescence to indicate expression of XRN1p, and a reduction in GFP fluorescence to indicate degradation of TMV.
      </p>
    </div>

    <script>
      AOS.init();
    </script>

  </body>
</html>
{{Stony_Brook/footer}}
